tools/sdk-ai-bots/azure-sdk-qa-bot-backend/README.md

```diff
@@ -0,0 +1,120 @@
+# Azure SDK QA Bot Backend
+
+The Azure SDK QA Bot Backend is a service that powers a conversational assistant for Microsoft Teams, specifically designed to help developers with TypeSpec-related questions. It leverages Azure's AI services to provide accurate and context-aware responses by searching through comprehensive TypeSpec documentation.
+
+## Overview
+
+This service integrates with Microsoft Teams and provides an intelligent chatbot interface that can:
+- Answer questions about TypeSpec syntax and usage
+- Provide code examples and best practices
+- Search through official TypeSpec documentation
+- Assist with TypeSpec-related troubleshooting
+
+## Features
+
+The bot provides intelligent responses by searching through knowledge bases including:
```

Comment:
there was a list of resources above. Maybe I understood wrong, but the bot tool was considering the type of documents too